components/Feed/StickyHeader.js

```diff
@@ -0,0 +1,79 @@
+import { Component } from 'react'
+import { HEADER_HEIGHT, HEADER_HEIGHT_MOBILE } from '../constants'
+import { css } from 'glamor'
+import { mediaQueries } from '@project-r/styleguide/lib/lib'
+
+const stickyHeaderStyle = {
```

Comment:
We typically name the default styles object just `styles` to avoid lengthy references like`stickyHeaderStyle.sticky`.